### PR TITLE
Update default-storage-class-prereqs.md

### LINKED ISSUE #29641: Improvement for k8s.io/docs/tasks/run-application/run-single-instance-stateful-application/
Please provide links on the 'Before you begin' section, teaching how to 'have dynamic PersistentVolume provisioner with a default StorageClass'.

I'm quite new to K8s and am not sure how to verify if I have the former half of this sentence, and the latter end of it (static provisioning) looks less automated / more difficult.

### PR DESCRIPTION
**Pull request by AvineshTripathi**
fixes #29641

### CODE REVIEW

File: content/en/includes/default-storage-class-prereqs.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-You need to either have a dynamic PersistentVolume provisioner with a default
+You need to either have a [dynamic PersistentVolume provisioning](docs/concepts/storage/dynamic-provisioning/) with a default
```
Comment (RinkiyaKeDad) [inline content/en/includes/default-storage-class-prereqs.md:1]:
```suggestion
You need to either have a [dynamic PersistentVolume provisioning](/docs/concepts/storage/dynamic-provisioning/) with a default
```

Comment (AvineshTripathi) [reply content/en/includes/default-storage-class-prereqs.md:1]: Hey thank you for reviewing I think provisioning is a better word here as here we are talking about the process and not the tool which does the process. Please correct me if I am wrong

**Comment by reylejano** [comment]
I agree with @RinkiyaKeDad in his [comment](https://github.com/kubernetes/website/pull/30835/files#r766536191) that the original is better.
I believe the proposed change is trying to use "provisioning" as a gerund (a verb typically ending in -ing used as a noun) but the original is less confusing and grammatically correct

Feel free to re-open if you think otherwise

/close

**Comment by AvineshTripathi** [comment]
Hey @reylejano the PR was mainly about adding link to provisioner so are we looking to add the link by keeping the word as it is in the original form

**Comment by reylejano** [comment]
@AvineshTripathi , I re-opened the PR, I agree with @RinkiyaKeDad 's [suggestion](https://github.com/kubernetes/website/pull/30835/files#r766536191)

**Comment by annajung** [comment]
Thanks @AvineshTripathi 

/lgtm
/hold until deployment passes
I believe @kubernetes/sig-docs-leads can retrigger the build

**Review by sftim — APPROVED** [review]
/approve

**Comment by sftim** [comment]
/unhold
preview is done

/approve

**Pull request merged**